architecture/additional_concepts/ephemeral-storage.adoc

```diff
@@ -0,0 +1,89 @@
+[[architecture-additional-concepts-ephemeral-storage]]
+= Ephemeral Local Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+*Note: this only applies if the ephemeral-storage tech preview in
+{product-title} 3.10 has been enabled.  This is disabled by default;
+to enable this features, please see
+xref:../install_config/configuring_ephemeral.adoc[Configuring for
+Ephemeral Storage].*
+
+== Overview
+
+In addition to persistent storage, pods and containers may require
+ephemeral (transient) local storage for their operation.  The lifetime
+of this ephemeral storage does not extend beyond the life of the
+individual pod, and this ephemeral storage cannot be shared across
+pods.
+
+Prior to {product-title} 3.10, ephemeral local storage was exposed to
+pods via the container’s writable layer, logs directory, and EmptyDir
+volumes.  Pods use ephemeral local storage for scratch space, caching
+and logs.  There are many issues related to the lack of local storage
+accounting and isolation, including:
+
+- Pods do not know how much local storage is available to them.
+
+- Pods cannot request "guaranteed" local storage.
+
+- Local storage is a "best effort" resource.
+
+- Pods can get evicted due to other pods filling up the local storage,
+after which no new pods will be admitted until sufficient storage
+has been reclaimed.
+
+Unlike persistent volumes, ephemeral storage is unstructured and
+shared (the space, not the actual data) between all pods running on a
+node, in addition to other uses by the system, the container runtime,
+and {product-title}.  The ephemeral storage framework allows pods to
+specify their transitory storage needs, and {product-title} to
+schedule pods where appropriate and protect the node against excessive
+use of local storage.
+
+While the ephemeral storage framework allows administrators and
+developers to better manage this resource, it does not provide any
+promises related to I/O throughput and latency, as this resource is
```

Comment:
I would just end the sentence at "latency."

Reply:
+1